### DISCUSSION TITLE
Make a carousel primitive

### DISCUSSION THREAD

**Discussion by deebov** [radix-ui/primitives#1932] [Ideas]
I've seen the carousel example on the marketing website. It works very good IMO. Are there any plans to make it a primitive component?

[Source of the carousel](https://github.com/radix-ui/website/blob/main/components/marketing/Carousel.tsx)

**Comment by andy-hook** [comment]
It's not in development right now but we know folks are asking for this, see discussion in https://github.com/radix-ui/primitives/issues/388
